src/Spice86.Core/Emulator/LoadableFile/Dos/Exe/ExeLoader.cs

```diff
@@ -75,13 +82,10 @@ public class ExeLoader : DosFileLoader {
     private void LoadExeFileInMemory(ExeFile exeFile, ushort startSegment) {
         uint physicalStartAddress = MemoryUtils.ToPhysicalAddress(startSegment, 0);
         _memory.LoadData(physicalStartAddress, exeFile.ProgramImage);
-        for (int i = 0; i < exeFile.RelocationTable.Count; i++) {
-            SegmentedAddress address = exeFile.RelocationTable[i];
+        foreach (SegmentedAddress address in exeFile.RelocationTable) {
             // Read value from memory, add the start segment offset and write back
             uint addressToEdit = MemoryUtils.ToPhysicalAddress(address.Segment, address.Offset) + physicalStartAddress;
-            int segmentToRelocate = _memory.UInt16[addressToEdit];
-            segmentToRelocate += startSegment;
-            _memory.UInt16[addressToEdit] = (ushort)segmentToRelocate;
+            _memory.UInt16[addressToEdit] += startSegment;
```

Comment:
There's no relocations anymore ?

Reply:
Maybe I don't see an issue you see, but there is, and I tested the change, the code is the same a before, what changed:
 - The foreach instead of for
 - Now that we have indexers everywhere we can do += instead of get and set

Reply:
I misread it on my phone. There's no issue here. :)